zeth.txt

```diff
@@ -0,0 +1,14 @@
+[Section]
+Name = Zeth
+Version = 0.0.1
+License = Freeware
+Description = Zeth Visual Style for ReactOS
+Size = 116 KiB
+Category = 15
+URLDownload = https://github.com/Mas4hmad/ms-visual-styles/raw/master/ZETH.msstyles
+SHA1 = D06F5C1ADB15FB892F07583133D02E23AC3575C7
+CDPath = none
+SizeBytes = 119.658 bytes
```

Comment:
```suggestion
SizeBytes = 119658
```

Reply:
This should just be a number, not any formatting

Reply:
thanks, will do correctly